admin_guide/monitoring.rst

```diff
@@ -46,10 +46,10 @@ Installation
 
 The monitoring stack is installed as part of the Terraform deployment process. The Checkmk agent is deployed as a global docker service.
 
-Slack Ingration
+Slack Integration
 ---------------
```

Comment:
Hi and welcome! Thanks for your 1st contribution! Could you add some `-` to align them with the header?

Reply:
Thanks @Xarthisius, all set